lib/mandrill_dm/message.rb

```diff
@@ -100,6 +104,10 @@ def return_path_domain
       return_string_value(:return_path_domain)
     end
 
+    def send_at
+      return_time_as_formatted_string(get_value(:send_at))
```

Comment:
I'd rather see a ternary here that returns `nil` here when `send_at` is nil rather than having `return_time_as_formatted_string` doing that.  `return_time_as_formatted_string` should only handle date, time, and datetime (also with the cleanups mentioned by @kshnurov).

```
def send_at
  value = get_value(:send_at)
  value ? return_time_as_formatted_string(value) : nil
end
```